Sources/Mustache/Template+Render.swift

```diff
@@ -28,7 +28,8 @@ extension MustacheTemplate {
         if let indentation = context.indentation, indentation != "" {
             for token in tokens {
                 let renderedString = self.renderToken(token, context: &context)
-                if renderedString != "", string.last == "\n" {
+                // if rendered string is not empty and we are on a new line
+                if renderedString.count > 0, string.last == "\n" {
```

Comment:
Wouldn't this express it more clearly?

```suggestion
                if !renderedString.isEmpty, string.last == "\n" {
```

Reply:
I prefer a test that doesn't include a !